doc/HttpLuaModule.wiki

```diff
@@ -2011,6 +2011,17 @@ When exceeding this limit, Nginx will stop running the callbacks of newly expire
 
 This directive was first introduced in the <code>v0.8.0</code> release.
 
+== lua_semaphore_threshold ==
+
+'''syntax:''' ''lua_semaphore_threshold <threshold>''
+
+'''default:''' ''lua_semaphore_threshold 100000''
+
+'''context:''' ''http''
+
+The threshold of lua semaphore.
```

Comment:
I'm afraid the "threshold" word is quite confusing. Do you mean the maximum count of thresholds allowed in any individual nginx worker? Or something else? Please be explicit.